src/components/play-page.js

```diff
@@ -122,6 +122,7 @@ export class FlashCards extends connect(store)(PageViewElement) {
       <a-card hidden?="${_showSettingsPage}"
         question="${_card.question}"
         answer="${_card.answer}"
+        _answers="${_card._answers}"
```

Comment:
We've been using `_` as a [convention](https://github.com/Polymer/pwa-starter-kit/wiki/3.-Configuring-and-personalizing#naming-and-code-conventions) to indicated private/protected variables, which means you wouldn't be setting them from the outside. This should probably be just `answers` :)